keps/prod-readiness/sig-network/2086.yaml

```diff
@@ -3,3 +3,5 @@ alpha:
   approver: "@wojtek-t"
 beta:
   approver: "@wojtek-t"
+stable:
+  approver: "@wojtek-t"
```

Comment:
@wojtek-t all the PRR questions are answered from Beta but please double check :)

Reply:
> Are there any tests for feature enablement/disablement?

Were the tests added? Can you please link them in that question?

> What specific metrics should inform a rollback?

Have the metrics for blackholing were added as mentioned there?

> Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?

Were the manual tests run? Can you describe the findings?

> How can an operator determine if the feature is in use by workloads?

Please resolve the TODO by adding metric name there.

Reply:
Tests were added here https://github.com/kubernetes/kubernetes/pull/103409 -- will update the KEP along with other questions you raised.

Reply:
> Have the metrics for blackholing were added as mentioned there?

We haven't added this yet, implementation ended up being trickier than we thought but we didn't think it warranted blocking the feature. Will look at this again this release.

Reply:
> We haven't added this yet, implementation ended up being trickier than we thought but we didn't think it warranted blocking the feature. Will look at this again this release.

If we don't have metrics, I don't think I'm in a position to approve that for GA, TBH.
I will discuss it a bit more with prod-readiness group...

Reply:
I added a link about the e2e tests we added and I added metrics to the GA graduation criteria.

Reply:
> I added a link about the e2e tests we added and I added metrics to the GA graduation criteria.

I think we should go the other way.
Let's add metrics in this release (and finish other things if there is anything else to be done) as part of second beta release.

And move GA to the 1.24 (where it will effectively be formality and just feature-gate change).

Reply:
yup I already removed the GA milestone in kep.yaml

Reply:
Oh oops I didn't update the PRR stable milestone though, let me do that